.github/workflow_templates/build-and-test_dev.yml

```diff
@@ -155,3 +155,32 @@ jobs:
       - pull_request_info
       - build_deckhouse
 {!{ tmpl.Exec "tests_template" (slice $ctx "validators" "build_deckhouse") | strings.Indent 4 }!}
+
+  set_e2e_requirement_status:
+    # if previous jobs were failed we do not need set status, because checks will be failed
+    if: ${{ success() && needs.pull_request_info.outputs.changes_not_markdown == 'true' }}
+    name: Set 'waiting for e2e' commit status
+    needs:
+      - git_info
+      - pull_request_info
+      - build_deckhouse
+      - validators
+      - openapi_test_cases
+      - golangci_lint
+      - dhctl_tests
+      - matrix_tests
+      - tests
```

Comment:
Can we set initial status right after git_info and pull_request_info? Is it vital to wait for build and tests?

Reply:
[same here](https://github.com/deckhouse/deckhouse/pull/2431#discussion_r970464240)